lib/diagnostic.rb

```diff
@@ -100,13 +105,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'pop(2)'
```

Comment:
you have to call what you want the `.pop` method to be applied to. `arr.pop(2)` but correct.